docs/artifacts/codex-programme-order.html

```
@@ -0,0 +1,135 @@
+<title>Fix Codex Before Asking It</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)